doc/gallery/introduction/pytensor_intro.ipynb

```diff
@@ -0,0 +1,3311 @@
+{
```

Comment:
<p>I recall that Ricardo had an objection to my quip about this graphic being "upside down"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pytensor/pull/1585/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Any suggested changes @ricardoV94 ?

Reply:
<p>Yes it's not upside down, it's how information flow. You can only follow those arrows, and not the other way around (i.e., you can know what apply an output variable comes from, but not in which apply an "input variable" is used (it could actually be several). Same for type, a variable has a type, but from a type you don't know which variable has it (again it could be several)... and so on.</p>